ServiceApp/src/org/droidplanner/services/android/core/gcs/location/FusedLocation.java

```diff
@@ -41,6 +41,7 @@
 
     private final GoogleApiClientManager gApiMgr;
     private final GoogleApiClientTask requestLocationUpdate;
+    private boolean mLocationUpdatesEnabled = false;
```

Comment:
@kellyschrock is this flag needed? Are there adverse effects to calling `disableLocationUpdates()` if `enableLocationUpdates()` hasn't been called before?